<ui version="4.0" >
 <class>ModelSettingsDlg</class>
 <widget class="QWidget" name="ModelSettingsDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Samplerate:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbSamplerate" >
       <property name="minimum" >
        <number>32000</number>
       </property>
       <property name="maximum" >
        <number>128000</number>
       </property>
       <property name="value" >
        <number>44100</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Hz</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Kanäle:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbChannels" >
       <property name="minimum" >
        <number>1</number>
       </property>
       <property name="maximum" >
        <number>8</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lbProcessinglist" >
     <property name="text" >
      <string>Bearbeitungsliste:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QTableWidget" name="twProcessingFilters" >
         <column>
          <property name="text" >
           <string>Befehl</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QToolButton" name="tbUp" >
           <property name="text" >
            <string>...</string>
           </property>
           <property name="icon" >
            <iconset resource="../simon.qrc" >:/images/icons/go-up.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tbDown" >
           <property name="text" >
            <string>...</string>
           </property>
           <property name="icon" >
            <iconset resource="../simon.qrc" >:/images/icons/go-down.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pbAdd" >
           <property name="text" >
            <string>Hinzufügen</string>
           </property>
           <property name="icon" >
            <iconset resource="../simon.qrc" >:/images/icons/list-add.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbRemove" >
           <property name="text" >
            <string>Entfernen</string>
           </property>
           <property name="icon" >
            <iconset resource="../simon.qrc" >:/images/icons/list-remove.svg</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QLabel" name="lbLegend" >
         <property name="text" >
          <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600; font-style:italic;">Legende:&lt;/span>&lt;/p>
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-weight:600; font-style:italic;">&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-weight:600;">%1:&lt;span style=" font-weight:400;"> Eingabedateiname&lt;/span>&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600;">%2:&lt;/span> Ausgabedateiname&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600;">%3:&lt;/span> Samplerate&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600;">%4:&lt;/span> Kanäle&lt;/p>&lt;/body>&lt;/html></string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_4" >
         <property name="text" >
          <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-style:italic;">Bitte benutzen Sie komplette Pfade oder stellen Sie sicher, das die Programme im Suchpfad liegen.&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../simon.qrc" />
 </resources>
 <connections/>
</ui>
